addons/mass_mailing/static/src/scss/themes/theme_default.scss

```diff
@@ -91,7 +91,6 @@ div.col:not([align]) {
     }
 
     a:not(.btn), .btn.btn-link {
-        font-weight: bold;
```

Comment:
I checked and it is. If we want to keep the default, we can do this:
```diff
diff --git a/addons/mass_mailing/views/themes_templates.xml b/addons/mass_mailing/views/themes_templates.xml
index e0e36268f07..45807e6be7a 100644
--- a/addons/mass_mailing/views/themes_templates.xml
+++ b/addons/mass_mailing/views/themes_templates.xml
@@ -24,6 +24,9 @@
             p, p > *, li, li > * {
                 color: #6c757d;
             }
+            a:not(.btn), .btn.btn-link {
+                font-weight: bold;
+            }
             hr {
                 border-top-color: #ced4da !important;
             }
```
But since the button sets to `bolder` and not `bold`, it's probably best to set the default to `bolder` (I don't think the difference with `bolder` is particularly noticeable).

Reply:
Changes done it is working.

Reply:
It was working when i updated the code in local. I checked it again now it's not working that's surprising. I'll update it.